R/btw_this.R

```diff
@@ -0,0 +1,64 @@
+btw_this <- function(x, ...) {
+  UseMethod("btw_this")
+}
```

Comment:
This will be exported and documented, and some shell methods could appear here. But I envision the `btw_this.{class}` methods living next to their implementations and being individually documented, c.f. `btw_this.environment()`.